Source/WebKit/UIProcess/API/glib/WebKitApplicationInfo.h.in

```diff
@@ -17,15 +17,26 @@
  * Boston, MA 02110-1301, USA.
  */
 
+#if PLATFORM(GTK)
 #if !defined(__WEBKIT2_H_INSIDE__) && !defined(WEBKIT2_COMPILATION)
 #error "Only <webkit2/webkit2.h> can be included directly."
 #endif
+#elif PLATFORM(WPE)
+#if !defined(__WEBKIT_H_INSIDE__) && !defined(WEBKIT2_COMPILATION)
+#error "Only <wpe/webkit.h> can be included directly."
+#endif
+#endif
 
 #ifndef WebKitApplicationInfo_h
 #define WebKitApplicationInfo_h
 
 #include <glib-object.h>
+
+#if PLATFORM(GTK)
 #include <webkit2/WebKitDefines.h>
+#elif PLATFORM(WPE)
+#include <wpe/WebKitDefines.h>
+#endif
 
```

Comment:
And same here, we could convert the includes automatically, converting webkit2/ to wpe/ for wpe. Or using something like `#include <@PLATFORM_INCLUDE_PREFIX@/Foo.h>`  or if that's to ugly we could simply use `#include "Foo.h"` and convert all includes not using `<>` to `<prefix/Foo.h>`.

Reply:
Hm, if we do `#include <@PLATFORM_INCLUDE_PREFIX@/Foo.h>` then we might be able to use configure_file(). Otherwise, we'd definitely need a script.

Reply:
Agreed to do this in a follow-up patch.

Reply:
Wound up doing it in this PR anyway.